source_gen/lib/src/output_helpers.dart

```diff
@@ -26,7 +26,7 @@ Stream<String> normalizeGeneratorOutput(Object value) {
         return e.trim();
       }
 
-      throw _argError(e);
+      throw _argError(e as Object);
```

Comment:
I would just do a `whereType<Object>()` above instead of the explicit null check and cast.

Reply:
`whereType` doesn't exist on Stream! That's what I did first!

CC @lrhn

Reply:
Oh it does in `stream_transform` I think, if we don't already depend on it though then this is fine.